potpie/core/neo4j.py

```
@@ -0,0 +1,200 @@
+
+    async def verify_connection(self) -> bool:
+        """Verify Neo4j connectivity.
+
+        Returns:
+            True if connection is successful
+
+        Raises:
+            Neo4jError: If connection fails
+        """
+        if not self._initialized or self._driver is None:
+            raise NotInitializedError("Neo4j manager not initialized")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)